### PR TITLE
Optional vpacket logging WIP

### PR DESCRIPTION
**Pull request by orbitfold**

### CODE REVIEW

**Comment by orbitfold** [comment]
Try with this PR @aoifeboyle, I think I fixed your thing.

**Comment by aoifeboyle** [comment]
Will do and will let you know. Thank you @orbitfold.

**Comment by aoifeboyle** [comment]
@orbitfold Working now, thank you.

**Comment by orbitfold** [comment]
Glad to hear it worked :)

File: tardis/montecarlo/src/cmontecarlo.c [hunk 1]
```diff
@@ -423,13 +423,14 @@ montecarlo_one_packet (storage_model_t * storage, rpacket_t * packet,
 		rpacket_get_energy (packet) * doppler_factor_ratio);
 	      rpacket_set_nu(&virt_packet,rpacket_get_nu (packet) * doppler_factor_ratio);
 	      reabsorbed = montecarlo_one_packet_loop (storage, &virt_packet, 1, mt_state);
+#ifdef WITH_VPACKET_LOGGING
 	      if ((rpacket_get_nu(&virt_packet) < storage->spectrum_end_nu) &&
```
Comment (unoebauer) [inline tardis/montecarlo/src/cmontecarlo.c]: The ifdef block is also activates/deactivates the setting of
```storage->spectrum_virt_nu[virt_id_nu]``` in line 455. This is crucial part of calculating and storing the virtual spectrum. This part should live outside the ifdef vpacketlogging block. See #445

**Pull request merged**